esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -881,10 +881,10 @@ def studentschedules(self, request, tl, one, two, module, extra, prog, onsite=Fa
             students = list(ESPUser.objects.filter(filterObj.get_Q(restrict_to_active=False)).distinct())
 
         students.sort()
-        return ProgramPrintables.get_student_schedules(request, students, prog, extra, onsite)
+        return ProgramPrintables.get_student_schedules(students, prog, extra, onsite)
 
     @staticmethod
-    def get_student_schedules(request, students, prog, extra='', onsite=False):
```

Comment:
``request`` is used in the function.

I only removed it locally because, IIRC, I had also temporarily removed the line that used ``request``, and was trying to make it easier for me to generate schedules from the command line. But that part of the change should be reverted.